ultraLatency.py

```diff
@@ -198,6 +198,7 @@
         oh.setTriggerSource(ohboard,options.gtx,0x5) # GBT, 0x0 for GTX
         pass
 
+    oh.setTriggerThrottle(ohboard, options.gtx, options.throttle)
```

Comment:
To me this is just asking for it to make a mistake a write an incorrect value. We know the default setting is more likely to work than a write, might be (probably) negligible but we do not know what our miswrite rate is yet. This is probably fine. So the throttle is systematically what a counter must reach before the next L1A is distributed?